Remove matchNames from Inver

cmap99 opened the pull request:
Setting the matchNames to Topaz in this situation is incorrect as Inver has not replaced Topaz it is its own separate brand. Topaz had been purchased by Circle K and they have rebranded old Topaz location to Circle K but I haven't added a matchName to Circle K as i'm not certain every location that was a Topaz is now a Circle K

LaoshuBaby commented:
I should apologize because I saw in taginfo that there are still a lot of Topaz and mistook this to be consistent with the rebranding scenario we often encounter 
:(

LaoshuBaby commented:
I search commit history and found it was added in #7702 

--
Currently a lot of elements in OSM data still named Topaz so weekly collection will let program create a empty Topaz entry everytime we run build script. @cmap99 

See: https://github.com/osmlab/name-suggestion-index/commit/bb082c6e367d9bcb2267f96a33daa99ccb0d2d98

![图片](https://user-images.githubusercontent.com/42690037/215317810-fd2a3e7e-d231-40b4-8962-b39fc789588b.png)

cmap99 commented:
> I should apologize because I saw in taginfo that there are still a lot of Topaz and mistook this to be consistent with the rebranding scenario we often encounter :(

That's okay I could've explained the situation better, or made two separate PRs.

Inver is its own brand that's been around for a while but was never added to NSI.

Topaz was its own brand but had been purchased by Circle K, and the majority of Topaz Fuel Stations were rebranded under the Circle K brand (but not all of them).

I have been reviewing the remaining locations still tagged as Topaz and correcting them. 



> I search commit history and found it was added in #7702
> 
> -- Currently a lot of elements in OSM data still named Topaz so weekly collection will let program create a empty Topaz entry everytime we run build script. @cmap99
> 
> See: [bb082c6](https://github.com/osmlab/name-suggestion-index/commit/bb082c6e367d9bcb2267f96a33daa99ccb0d2d98)
> 
> ![图片](https://user-images.githubusercontent.com/42690037/215317810-fd2a3e7e-d231-40b4-8962-b39fc789588b.png)

@LaoshuBaby Is there a need for a new PR to removed this once I've corrected the situation and replaced the Topaz brand?
